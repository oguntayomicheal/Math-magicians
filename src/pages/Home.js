function Home() {
  return (
    <div>
      <h2>Welcome to our Page!</h2>
      <p>
        lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor
        lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Dignissimos
        porro quis nemo dolore autem unde voluptatum nostrum vel nobis culpa
        aliquam totam temporibus atque accusantium blanditiis sunt officiis
        sequi, libero adipisci voluptates et eum incidunt quos corrupti. Modi
        optio illo nulla aut at tempore corporis unde tempora, vero quasi vel.
      </p>
    </div>

  );
}

export default Home;
