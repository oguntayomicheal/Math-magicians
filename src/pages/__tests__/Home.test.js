import { render } from '@testing-library/react';
import Home from '../Home';

it('Home renders as expected', () => {
  const homePage = render(<Home />);
  expect(homePage).toMatchSnapshot();
});
