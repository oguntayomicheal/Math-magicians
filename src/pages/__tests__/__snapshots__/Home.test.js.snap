// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders as expected 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <h2>
          Welcome to our Page!
        </h2>
        <p>
          lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor Lorem ipsum dolor sit amet consectetur adipisicing elit. Dignissimos porro quis nemo dolore autem unde voluptatum nostrum vel nobis culpa aliquam totam temporibus atque accusantium blanditiis sunt officiis sequi, libero adipisci voluptates et eum incidunt quos corrupti. Modi optio illo nulla aut at tempore corporis unde tempora, vero quasi vel.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <h2>
        Welcome to our Page!
      </h2>
      <p>
        lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor Lorem ipsum dolor sit amet consectetur adipisicing elit. Dignissimos porro quis nemo dolore autem unde voluptatum nostrum vel nobis culpa aliquam totam temporibus atque accusantium blanditiis sunt officiis sequi, libero adipisci voluptates et eum incidunt quos corrupti. Modi optio illo nulla aut at tempore corporis unde tempora, vero quasi vel.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
